core/src/main.cpp

```diff
@@ -651,21 +676,38 @@ auto set_log_level(const json& _config) -> void
 
 auto init_tls(const json& _config) -> void
 {
-	const auto set_env_var = [&](const auto& _json_ptr_path, const char* _env_var, const char* _default_value = "") {
-		using json_ptr = json::json_pointer;
+	const auto set_env_var_string =
+		[&](const auto& _json_ptr_path, const char* _env_var, const char* _default_value = "") {
```

Comment:
Just to be sure, the capture only takes `_config` right?

Reply:
Correct.